<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myCalcClass</class>
 <widget class="QWidget" name="myCalcClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>490</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>490</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>490</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>计算器</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton
{
	border-radius:10px;
	background-color: rgb(255, 255, 255);
}

QPushButton#btn1,#btn2,#btn3,#btnCheng,#btnChu,#btnDEL,#btnDeng,#btnJia,#btnJian,#btnKaiFang,#btnPingFang,#btnXFenZhiYi
{
	background-color: rgb(246,249,252);
}

QPushButton#btnDeng
{
	background-color: rgb(0,103,192);
	color: rgb(255, 255, 255);
}

QPushButton:hover
{
	background-color: rgb(251,252,253);
}

QPushButton#btn1:hover,#btn2:hover,#btn3:hover,#btnCheng:hover,#btnChu:hover,#btnDEL:hover,#btnDeng:hover,#btnJia:hover,#btnJian:hover,#btnKaiFang:hover,#btnPingFang:hover,#btnXFenZhiYi:hover
{
	background-color: rgb(243,247,250);
}

QPushButton#btnDeng:hover
{
	background-color: rgb(24,117,197);
	color: rgb(255, 255, 255);
}

</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>491</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="3" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="line2">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>370</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>28</pointsize>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background:transparent;
border-width:0;
border-style:outset</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QLabel" name="line1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(95,98,100);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="buttons">
      <property name="bottomMargin">
       <number>7</number>
      </property>
      <property name="horizontalSpacing">
       <number>3</number>
      </property>
      <property name="verticalSpacing">
       <number>4</number>
      </property>
      <item row="2" column="2">
       <widget class="QPushButton" name="num9">
        <property name="minimumSize">
         <size>
          <width>92</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>92</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="btnDeng">
        <property name="minimumSize">
         <size>
          <width>92</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>92</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="num7">
        <property name="minimumSize">
         <size>
          <width>92</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>92</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="num4">
        <property name="minimumSize">
         <size>
          <width>92</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>92</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btnXFenZhiYi">
        <property name="minimumSize">
         <size>
          <width>92</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>92</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>1/x</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="num6">
        <property name="minimumSize">
         <size>
          <width>92</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>92</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="btnDEL">
        <property name="minimumSize">
         <size>
          <width>92</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>92</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>DEL</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btnKaiFang">
        <property name="minimumSize">
         <size>
          <width>92</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>92</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>√x</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="btn1">
        <property name="minimumSize">
         <size>
          <width>92</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>92</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnPingFang">
        <property name="minimumSize">
         <size>
          <width>92</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>92</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>x²</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="btnChu">
        <property name="minimumSize">
         <size>
          <width>92</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>92</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>÷</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="num0">
        <property name="minimumSize">
         <size>
          <width>92</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>92</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="num2">
        <property name="minimumSize">
         <size>
          <width>92</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>92</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="num1">
        <property name="minimumSize">
         <size>
          <width>92</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>92</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="num3">
        <property name="minimumSize">
         <size>
          <width>92</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>92</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btn3">
        <property name="minimumSize">
         <size>
          <width>92</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>92</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="btnJian">
        <property name="minimumSize">
         <size>
          <width>92</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>92</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="btnCheng">
        <property name="minimumSize">
         <size>
          <width>92</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>92</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>×</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="btnJia">
        <property name="minimumSize">
         <size>
          <width>92</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>92</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="num8">
        <property name="minimumSize">
         <size>
          <width>92</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>92</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btn2">
        <property name="minimumSize">
         <size>
          <width>92</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>92</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="numSign">
        <property name="minimumSize">
         <size>
          <width>92</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>92</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>17</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>±</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="numDian">
        <property name="minimumSize">
         <size>
          <width>92</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>92</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>17</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="num5">
        <property name="minimumSize">
         <size>
          <width>92</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>92</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myCalc.qrc"/>
 </resources>
 <connections/>
</ui>
